src/constants/Services/ServicesData.js

```diff
@@ -0,0 +1,23 @@
+import web from '../../assets/images/services/web_development.jpg';
+import mobile from '../../assets/images/services/mobile_app.jpg';
+import desktop from '../../assets/images/services/desktop_app.jpg';
+
+
+export const ServicesData = [ 
+    {
+        img: web,
+        title: 'Web App Development',
```

Comment:
Remove your constants folder and transfer your Service and Navbar Data to our utils helper.js